openpype/hosts/fusion/plugins/publish/collect_render.py

```diff
@@ -0,0 +1,211 @@
+import os
+import attr
+import pyblish.api
+
+from openpype.pipeline import publish
+from openpype.pipeline.publish import RenderInstance
+from openpype.hosts.fusion.api.lib import get_frame_path
+
+
+@attr.s
+class FusionRenderInstance(RenderInstance):
+    # extend generic, composition name is needed
+    fps = attr.ib(default=None)
+    projectEntity = attr.ib(default=None)
+    stagingDir = attr.ib(default=None)
+    app_version = attr.ib(default=None)
+    tool = attr.ib(default=None)
+    workfileComp = attr.ib(default=None)
+    publish_attributes = attr.ib(default={})
+
+
+class CollectFusionRender(
+    publish.AbstractCollectRender,
+    publish.ColormanagedPyblishPluginMixin
+):
+
+    order = pyblish.api.CollectorOrder + 0.09
+    label = "Collect Fusion Render"
+    hosts = ["fusion"]
+
+    def get_instances(self, context):
+
+        comp = context.data.get("currentComp")
+        comp_frame_format_prefs = comp.GetPrefs("Comp.FrameFormat")
+        aspect_x = comp_frame_format_prefs.get("AspectX")
+        aspect_y = comp_frame_format_prefs.get("AspectY")
+
+        instances = []
+        instances_to_remove = []
+
+        current_file = context.data["currentFile"]
+        version = context.data["version"]
+
+        project_entity = context.data["projectEntity"]
+
+        for inst in context:
+            if not inst.data.get("active", True):
+                continue
+
+            family = inst.data["family"]
+            if family != "render":
+                continue
+
+            task_name = inst.data.get("task")  # legacy
+            tool = inst.data["transientData"]["tool"]
+
+            instance_families = inst.data.get("families", [])
+            subset_name = inst.data["subset"]
+            instance = FusionRenderInstance(
+                family="render",
+                tool=tool,
+                workfileComp=comp,
+                families=instance_families,
+                version=version,
+                time="",
+                source=current_file,
+                label="{} - {}".format(subset_name, family),
```

Comment:
This is quite an unfortunate labeling because due to this it loses the frame range related part of the label collected in Collect Instances. It'd be nice if we could preserve that label. E.g. `inst.data.get("label", "{} - {}".format(subset_name, family))` or alike.

Currently I couldn't find a way to quickly see for the instances what frame ranges are to be rendered.

For example, this to me is much more informative:
_This is `{subset} ({frameStart}-{frameEnd})`_ (with this label override replaced with the suggestion below):
![image](https://github.com/ynput/OpenPype/assets/2439881/f79d4af3-f72b-4766-894a-f57ffedf00ad)


_This is `{subset} ({frameStart}-{frameEnd}) [{handleStart}-{handleEnd}]`_ (with some changes in Collect Render):
![image](https://github.com/ynput/OpenPype/assets/2439881/a1b07ca8-d395-4954-a62c-16acffbd121b)

Than this:
![image](https://github.com/ynput/OpenPype/assets/2439881/fdc0c8ce-845e-4843-9668-9fbf4fcccb71)

Especially because it's pretty clear that the family is `render` since it's a Render instance.

So:
```suggestion
                label=inst.data.get("label",
                                    "{} - {}".format(subset_name, family)),
```

It's also quite an annoying downside of the new publisher that it doesn't trigger some sort of precollect stage which could show me the resulting frame range when changing the attributes, etc. I'd love to be able to just see that it's going to publish frame X to Y before I click publish.